Lesson14/fercan/script.js

```diff
@@ -0,0 +1,28 @@
+// 9. Declare a variable number6 and initialize it to 6. Declare a variable number7 and initialize it to `++number6` (number6 incremented using prexif ++ operator). Log both values to the console.
+let number6 = 6 ;
+number7 = '++number6' ;
```

Comment:
You need to declare a variable number7. Proper way of doing it is to use either 'let' or 'const' keyword in the name of the variable.
Sice we want to set number7 to number6 + 1, don't use quotes around `++number6`.
```suggestion
const number7 = ++number6;
```